docs/buildrun.md

```diff
@@ -132,6 +133,24 @@ You can also use set the `spec.serviceAccount.generate` path to `true`. This wil
 
 _**Note**_: When the SA is not defined, the `BuildRun` will default to the `default` SA in the namespace.
 
+## Canceling a `BuildRun`
+
+To cancel a `BuildRun` that's currently executing, update its status to mark it as canceled.
+
+When you cancel a `BuildRun`, the underlying `TaskRun` is marked as canceled per the [Tekton cancel `TaskRun` feature](https://github.com/tektoncd/pipeline/blob/main/docs/taskruns.md).
+
+Example of canceling a `BuildRun`:
+
+```yaml
+apiVersion: shipwright.io/v1alpha1
+kind: BuildRun
+metadata:
+  name: buildpack-nodejs-buildrun-namespaced
+spec:
+  # [...]
+  stateOverride: "BuildRunCanceled"
```

Comment:
I think that this is supposed to just be `state`

Reply:
yep missed this in the latest name change

Reply:
updated / fixed